File: nodejs/01-estrutura-sequencial/ex10.js
/*
 * Exercício 10
 *
 * Faça um Programa que peça a temperatura em graus Celsius, transforme e mostre em graus
 * Fahrenheit.
 */

import * as readlinePromises from 'node:readline/promises'
import {stdin as input, stdout as output} from 'node:process'

const rlp = readlinePromises.createInterface({input, output})

const celsius = parseFloat(await rlp.question("Celsius: "))
rlp.close()

const fahrenheit = parseFloat(celsius * 9 / 5 + 32).toFixed(2)

console.log(`A temperatura convertida em Fahrenheit é: ${fahrenheit} graus`)

File: python/01_estrutura_sequencial/ex10.py
"""
Exercício 10

Faça um Programa que peça a temperatura em graus Celsius, transforme e mostre em graus
Fahrenheit.
"""

celsius = float(input("Celsius: "))
fahrenheit = round(celsius * 9 / 5 + 32, 2)
print(f"A temperatura convertida em Fahrenheit é: {fahrenheit} graus")
